language/de-DE/tpl_cassiopeia.ini

```diff
@@ -6,6 +6,7 @@
 
 TPL_CASSIOPEIA_XML_DESCRIPTION="Weiterentwicklung der Space-Themes (Solarflare in 1.0, Milkyway in 1.5 und Protostar in 3.0), Cassiopeia ist das Joomla 4 Frontend Template basierend auf Bootstrap 4."
 
+CASSIOPEIA="Cassiopeia Seitentemplate"
```

Comment:
```suggestion
CASSIOPEIA="Cassiopeia Frontend Template"
```

Oder?

Reply:
Dann müssen wir das in der sys.ini aber nachziehen.

Reply:
würde ich vorschlagen oder in de XML_DESCRIPTION